spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaReactiveWebServerFactory.java

```diff
@@ -207,7 +217,27 @@ public WebServer getWebServer(HttpHandler httpHandler) {
                 firstNonNull(findBean(DataBufferFactoryWrapper.class), DataBufferFactoryWrapper.DEFAULT);
 
         final Server server = configureService(sb, httpHandler, factoryWrapper, getServerHeader()).build();
-        return new ArmeriaWebServer(server, protocol, address, port, beanFactory);
+        final ArmeriaWebServer armeriaWebServer = new ArmeriaWebServer(server, protocol, address, port,
+                                                                       beanFactory);
+        if (!isManagementPortEqualsToServerPort()) {
+            // Since this method will be called twice, need to reuse ArmeriaWebServer
```

Comment:
How about leaving a comment?
`The management port is set to the Server in ArmeriaSpringActuatorAutoConfiguration.`